buildchain/buildchain/codegen.py

```diff
@@ -128,6 +128,7 @@ def codegen_chart_fluent_bit() -> types.TaskDict:
 def codegen_chart_ingress_nginx() -> types.TaskDict:
     """Generate the SLS file for NGINX Ingress using the chart render script."""
     chart_dir = constants.CHART_ROOT / "ingress-nginx"
+    namespace = "metalk8s-ingress"
```

Comment:
This is a constant value, why not declaring it as a constant on top of the file / in the constants package ?